### PR TITLE
adding python utils

### PR DESCRIPTION
**Pull request by vevinkirk**
Some python utility functions

### CODE REVIEW

File: src/api/python/league_util.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
+LEAGUE_API_Key="something"
```
Comment (cannedyeti) [inline src/api/python/league_util.py]:
```suggestion
LEAGUE_API_KEY="something"
```

nit

File: src/api/python/league_util.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+def league_test():
+    api_key = LEAGUE_API_Key
```
Comment (cannedyeti) [inline src/api/python/league_util.py]:
```suggestion
    api_key = LEAGUE_API_KEY
```

File: src/api/python/league_util.py [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+    watcher = LolWatcher(api_key)
+    my_region = 'na1'
+    me = watcher.summoner.by_name(my_region, "virkthejerk")
+    await ctx.send((me))
+
+def get_league_rank(summoner):
+    api_key = LEAGUE_API_Key
```
Comment (cannedyeti) [inline src/api/python/league_util.py]:
```suggestion
    api_key = LEAGUE_API_KEY
```

File: src/api/python/league_util.py [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+    watcher = LolWatcher(api_key)
+    my_region = 'na1'   
+    me = watcher.summoner.by_name(my_region, summoner)
+    my_ranked_stats = watcher.league.by_summoner(my_region, me['id'])
+    json_formatted_str = json.dumps(my_ranked_stats, indent=2)
+    response = f"```{json_formatted_str}```"
+    return response
+
+def get_league_rank_internal(summoner):
+    api_key = LEAGUE_API_Key
```
Comment (cannedyeti) [inline src/api/python/league_util.py]:
```suggestion
    api_key = LEAGUE_API_KEY
```

**Review by cannedyeti — APPROVED** [review]

**Review by brycerobinsn — APPROVED** [review]

**Pull request merged**